apiserver/facades/agent/storageprovisioner/service.go

```diff
@@ -291,6 +291,18 @@ type StorageProvisioningService interface {
 		ctx context.Context, volumeID string, unitUUID coreunit.UUID,
 	) (storageprovisioning.VolumeAttachmentUUID, error)
 
+	// GetVolumeParams returns the volume params for the supplied uuid.
```

Comment:
For consistency, we can just copy the comment(including errors) from the service methods.